cmd/notation/plugin.go

```diff
@@ -36,6 +44,44 @@ Example - List installed Notation plugins:
 	}
 }
 
+func pluginInstallCommand() *cobra.Command {
+	var force bool
```

Comment:
Should we make a struct instead of a single variable?

Reply:
Are there any other config settings we'd like to add to the struct? If so, it might be worthwhile to look into adding [viper](https://github.com/spf13/viper) for configuration.